core/config/project_settings.cpp

```diff
@@ -1314,10 +1314,6 @@ ProjectSettings::ProjectSettings() {
 
 	GLOBAL_DEF(PropertyInfo(Variant::STRING, "editor/script/templates_search_path", PROPERTY_HINT_DIR), "res://script_templates");
```

Comment:
What about this one, and the two just above?

Reply:
Probably a good idea to keep them all together yeah

Reply:
Any others you can see before I push this change?